test/over_react/component/fixtures/context_provider_component.dart

```diff
@@ -27,7 +27,7 @@ class _$ContextProviderWrapperProps extends UiProps {}
 
 @State()
 class _$ContextProviderWrapperState extends UiState {
-  int? latestValue;
+  late int latestValue;
```

Comment:
Changed this to non-nullable because it was being set to `Provider()..value` below, which only allowed non-nullable values:

```dart
return (counterContext.Provider()..value = this.state.latestValue)(
```